docs/env-variables.md

```diff
@@ -0,0 +1,20 @@
+# Different Environment Variables used
```

Comment:
Maybe we should also describe about `KUBECONFIG`?
like this.
```
`KUBECONFIG`:The cluster from which you are importing is indicated by this environment variable.
  [Read more](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable)
```

Reply:
Done with the latest push. Thanks for mentioning that.